setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
-name = linuxforhealth-{library name}
-version = attr: linuxforhealth.{library name}.__version__
```

Comment:
The pip/setup process parses this string for version info. I think the placeholder test gave us "the business" so I just updated it to use "fhir_utils".